src/audio/component.c

```diff
@@ -543,14 +543,18 @@ bool update_peak_of_measured_cpc(struct comp_dev *dev, size_t measured_cpc)
 bool comp_update_performance_data(struct comp_dev *dev, uint32_t cycles_used)
 {
 	struct perf_data_item_comp *item = dev->perf_data.perf_data_item;
-	/* we divide by ibs so we need to check if its set */
-	if (item && dev->ibs != 0) {
-		item->total_iteration_count++;
-		item->total_cycles_consumed += cycles_used;
-		item->item.avg_kcps = item->total_cycles_consumed * dev->ll_chunk_size_
```

Comment:
this code comes from an earlier commit in this PR, right? Why not use the final version immediately?